boards/posix/doc/board.rst

```diff
@@ -0,0 +1,158 @@
+In reality, what happens is that the timer will monitor how much real time
+has actually passed since boot, and if necessary will stall the exection
+before raising each timer interrupt.
+As, normally, the zephyr application and HW models run in very little time
+in the host CPU, this is a good enough approach.
+
+
+**Interrupt controller**
+
+A simple yet generic interrupt controller is provided. It can nest interrupts
+and provides interrupt priorities. Interrupts can be individually masked /
+unmasked. SW interrupts are also supported.
+
+
+Important limitations
+=====================
+
+The assumption that simulated time can only pass while the CPU is sleeping means
+that there cannot be busy wait loops in the application code which wait for
+something to happen without leting the CPU sleep. If this exist, they will
+behave as infinite loops which will stall the application.
+
+As simulated time does not pass while the CPU is running, it also means no HW
+interrupts will interrupt the threads execution unless the SW enables/unmasks
+them.
+This is intentional, as it provides a deterministic environment to develop and
+debug.
+But note that it may hide issues in the SW that may only be triggered in the
+real platform.
+
+That is, this native port of Zephyr is not meant and could not possibly help
+debug races between HW and SW, or similar timing related issues.
+
+
+How to use it
+*************
+
+Compiling
+=========
+
+   .. note::
+
+    Remember that you must have the 32bit C library installed in your system
+    (in Ubuntu 16.04 you can just install the gcc-multilib package)
+
+To compile the application in this way, just target this board. For example :
+
+.. code-block:: console
+
+   $ cd $ZEPHYR_BASE/samples/hello_world
+   $ mkdir build && cd build
+   $ cmake -DBOARD=simple_process ..
+   $ make
+
+
+Running
+=======
+
+The result of the compilation is an executable (zephyr.exe) placed in the
+zephyr/ subdirectory of the build folder.
+This executable can be run like any other Linux application.
+
+.. code-block:: console
+
+   $ zephyr/zephyr.exe
+   # Press Ctrl+C to exit
+
+Note that the Zephyr kernel does not actually exit once the application is
+finished. It simply goes into the idle loop forever.
+Therefore you must stop the application manually (Ctrl+C in Linux).
+
+For the tests which use ztest, the application will actually exit once all
+tests have been completed.
+
+If you want your application to gracefully finish when it reaches some point,
+you may add a conditionally compiled (CONFIG_BOARD_SIMPLE_PROCESS) call to
+main_clean_up(exit_code);
+
+
+Debugging
+=========
+
+As the resulting executable is a native application it can be debuged and
+instrumented like any other native program. The program is compiled with debug
+information, so it can be run directly in for example gdb or instrumented with
+valgrind.
+
+Note that as the execution is fully deterministic (there is no asynchronous or
```

Comment:
Change to:

    Because the execution of your Zephyr application is fully deterministic
    (there are no asynchronous or random components), you can execute the
    code multiple times and get the exact same result and instrumenting the
    code does not affect the execution of the code.